### DISCUSSION TITLE
Cloudlog Install wizard

### DISCUSSION THREAD

**Discussion by M0NGZ** [magicbug/Cloudlog#2281] [Q&A] [answered]
I'm following the guide for an install on a Raspberry Pi, but unsure about the Cloudlog Install Wizard. In the guide notes it shows: 
You need to run the install wizard. At this point, please open <url-to-cloudlog>/install and follow the guide. Is this a link that needs updating or something else?

Thanks in advance

Steve

**Comment by AndreasK79** [comment, marked as answer]
Hi.

Just add /install to the Cloudlog url. You will be asked to set up a few things.
